content/topics/how-to-ask-for-help-with-your-code/_index.md

```diff
@@ -33,22 +33,22 @@ staff member: Here is an explanation [explanation], and here are some resources
 #### Where to ask good questions
 
 - [Stackoverflow - How to ask for help](https://stackoverflow.com/help/how-to-ask) == TRUELY **AMAZING**
-- Any tech community discussion that you are a part of
+- Any tech community discussion that you are a part of(Discord, RocketChat)
 
 #### Characteristics of a good question:
 
-As a general rule: If you really want help then make it as easy as possible for people to help you. Empathise with the person who will help you. If they are left guessing or they have to ask a million follow up questions then that means you asked a bad-quality question.
+As a general rule: If you want help then make it as easy as possible for people to help you. Empathize with the person who will help you. If they are left guessing or they have to ask a million follow-up questions then that means you asked a bad-quality question.
```

Comment:
change Empathize to British English

Reply:
Grammarly doesn't suggest a change. it recognises the word empathize in British English so I'm not sure lol